src/app-gocardless/banks/commerzbank_cobadeff.js

```diff
@@ -0,0 +1,87 @@
+import Fallback from './integration-bank.js';
+import { amountToInteger, printIban } from '../utils.js';
+import { formatPayeeName } from '../../util/payee-name.js';
+
+/** @type {import('./bank.interface.js').IBank} */
+export default {
+  ...Fallback,
+
+  institutionIds: ['COMMERZBANK_COBADEFF'],
+
+  accessValidForDays: 179,
+
+  normalizeAccount(account) {
+    return {
+      account_id: account.id,
+      institution: account.institution,
+      mask: account.iban.slice(-4),
+      iban: account.iban,
+      name: [account.name, printIban(account)].join(' '),
+      official_name: account.product,
+      type: 'checking',
+    };
+  },
```

Comment:
As this is the same as in the [fallback](https://github.com/actualbudget/actual-server/blob/master/src/app-gocardless/banks/integration-bank.js) you could remove this and rely on the fallback

```suggestion
```